spec/webauthn/authenticator_attestation_response_spec.rb

```diff
@@ -438,4 +438,31 @@
       end
     end
   end
+
+  describe "attestation statement verification" do
+    let(:original_challenge) do
+      Base64.strict_decode64(seeds[:security_key_direct][:credential_creation_options][:challenge])
+    end
+
+    let(:origin) { "http://localhost:3000" }
+
+    let(:attestation_response) do
+      response = seeds[:security_key_direct][:authenticator_attestation_response]
+
+      WebAuthn::AuthenticatorAttestationResponse.new(
+        attestation_object: Base64.strict_decode64(response[:attestation_object]),
+        client_data_json: Base64.strict_decode64(response[:client_data_json])
+      )
+    end
+
+    before do
+      WebAuthn.configuration.verify_attestation_statement = true
+    end
+
+    context "when verification is set to false" do
```

Comment:
Description says "when set to false" but we're actually setting to `true`. Was that intentional?

To cover this feature we probably would need two scenarios:

Scenario 1:
Given attestation statement is invalid
And verify_attestation_statement config is set to `true`
Then verification fails

Scenario 2:
Given attestation statement is invalid
And verify_attestation_statement config is set to `false`
Then verification succeeds

Reply:
That was not intentional. I've fixed the error in the specs, and updated them to include both the scenarios.